### PR TITLE
feat(machine-api): stream a claimed-user authentication test as NDJSON events

### PR DESCRIPTION
**Pull request by archledger**
First of the three surfaces left in #108, and the foundation the other two need. Refs #108.

## What this adds

`irlume auth test --events=jsonl`, under a new `auth-test-events` capability. It answers whether the claimed account's live face matches its own enrolment:

```
{"contract_version":1,"engine_version":"0.7.1","command":"auth.test","operation_id":"8db4…","session_id":"a14a…","sequence":0,"event":"started","terminal":false,"data":{"operation":"auth-test"}}
{"contract_version":1,…,"sequence":1,"event":"capturing","terminal":false,"data":{}}
{"contract_version":1,…,"sequence":2,"event":"result","terminal":true,"data":{"granted":false,"live":true,"reason":"no-match"}}
```

## Security decisions worth reviewing

**The match score is withheld.** `AuthResult` carries a `score: f32`, and a caller that can read a continuous score can hill-climb against it, adjusting a presentation until it crosses the threshold. That turns a diagnostic into an oracle. `granted` and `live` are what a settings panel needs, and `reason` is derived from those two booleans, never from daemon prose, so a reworded message is not a breaking change.

**Cross-account probing is already refused by the daemon**, whose own comment names this risk. Verified rather than assumed: `auth test --user root` as an unprivileged caller exits 1 with a terminal error event. It reports `operation-failed`, not `not-authorized`, because distinguishing them would answer "does this account exist and is it enrolled" for any local process.

**Nothing is released.** `Authenticate` returns a verdict; credential release is a separate privileged path this command never touches. It also cannot change a profile or a threshold, because it sends no request that could.

**No username is echoed**, matching the existing rule that machine output carries no usernames. The caller already knows what it asked about, and a desktop may log the stream.

## Why this is additive, not contract 2

The contract says machine stdout contains exactly one JSON document. A stream emits many. That is compatible because a consumer reaches a streaming command only by invoking a capability it found in `capabilities` — so a contract 1 consumer written before streaming existed still observes one document from every command it knows how to call. Bumping to contract 2 would force the existing consumer to adopt a new contract for a purely additive feature. The docs now scope the single-document rule to non-streaming commands.

## Stream guarantees

Enforced by the emitter rather than by callers remembering to maintain them:

- `sequence` starts at zero, increments by one, no gaps — a lost line is detected by arithmetic, not a timeout
- exactly one event has `terminal` true and it is last — a reader knows when to stop
- `operation_id` is identical across the stream

`finish` and `fail` consume the stream, so a second terminal line is not expressible.

## One session per user

An advisory lock in the caller's runtime directory, not a recorded process id: the kernel releases it when the holder exits for any reason, so a panel killed mid-capture cannot strand the user. A second concurrent invocation gets a retryable `session-busy`. Verified with two concurrent invocations.

`--preview` is refused rather than accepted and ignored — accepting it would suggest frames were withheld by policy rather than never implemented.

## The capability ships with its paperwork

Docs, schema (`$defs/event`, `$defs/authTestResultData`), a fixture captured from a real engine, and conformance checks.

The conformance script needed care. An unlisted capability falls into its `this script does not know that capability` skip, which reads as a pass — the same silent-green shape as #163. Streaming capabilities are now listed explicitly and checked on the paths that need no camera: the command exists, negotiates a contract, and refuses a bad invocation as one document with exit 2 rather than starting a stream.

`.ndjson` fixtures were likewise invisible to the fixture loader, which matches only `*.json`. They now have their own check.

## Validation

- 765 workspace tests pass, clippy and `cargo fmt --check` clean
- conformance 27 passed, 0 failed, **0 skipped**
- the stream-fixture checks were verified by mutation rather than assumed: breaking the sequence, the terminal rule, the operation id, or emptying the file each fails with its own message, and the restored fixture passes
- exercised against the live daemon on real hardware, including a granted result, a `no-match`, a refused cross-account probe, and two concurrent sessions

## Not included

`enroll --events=jsonl --preview=ir-jpeg` and the login plan/apply/verify/rollback transactions remain unimplemented, and `docs/MACHINE-API.md` still lists them as outside contract 1. The preview path exports biometric imagery beyond irlume's boundary and wants its own threat-model entry rather than being folded in here.

### CODE REVIEW

File: crates/irlume-cli/src/machine.rs [hunk 1]
```diff
@@ -479,6 +598,183 @@ pub fn login_status(args: &[String]) -> ExitCode {
...
+                // SAFETY: getuid cannot fail and touches no memory.
+                std::path::PathBuf::from(format!("/tmp/irlume-{}", unsafe { libc::getuid() }))
+            });
+        let _ = std::fs::create_dir_all(&dir);
+        let file = std::fs::OpenOptions::new()
+            .create(true)
+            .truncate(false)
+            .write(true)
+            .open(dir.join("machine-session.lock"))
+            .ok()?;
```
[the remark this answers is not in the document]
Comment (archledger) [reply crates/irlume-cli/src/machine.rs]: Confirmed and fixed in 6e892ab. `.ok()?` collapsed both cases, so an unwritable runtime directory was reported as retryable `session-busy` and a consumer would spin against a permission error. `acquire` now returns a typed refusal: EWOULDBLOCK is another session and stays retryable, anything else is `operation-failed` and is not. Verified on both paths rather than reasoned about: a genuinely held lock still gives `session-busy`, and `XDG_RUNTIME_DIR=/proc/nonexistent-and-unwritable` gives `operation-failed, retryable=false`.

File: scripts/machine-api-conformance.py [hunk 2]
```diff
@@ -302,6 +371,58 @@ def check_fixtures(results, fixtures_dir, validate, validate_note):
...
+        if sequences != list(range(len(lines))):
+            results.fail(what, f"sequence must start at 0 and be gapless, got {sequences}")
+            continue
+        terminals = [bool(line.get("terminal")) for line in lines]
+        if terminals.count(True) != 1 or not terminals[-1]:
+            results.fail(what, "exactly one event must be terminal, and it must be last")
+            continue
+        if len({line.get("operation_id") for line in lines}) != 1:
+            results.fail(what, "every line must carry the same operation_id")
+            continue
```
[the remark this answers is not in the document]
Comment (archledger) [reply scripts/machine-api-conformance.py]: Confirmed and fixed in 6e892ab. `len({None}) == 1`, checked directly, so a capture where no line carried an `operation_id` passed the equality test. The check now requires the field present and non-empty on every line before comparing. Proved by mutation: stripping `operation_id` from every line now fails with `every line must carry a non-empty operation_id`, where it previously passed.

File: schemas/machine-api-v1.schema.json [hunk 3]
```diff
@@ -323,15 +594,116 @@
...
+          "description": "True on exactly one line, always the last."
+        },
+        "data": {
+          "type": "object"
+        },
+        "error": {
+          "$ref": "#/$defs/error"
+        }
+      }
+    },
```
[the remark this answers is not in the document]
Comment (archledger) [reply schemas/machine-api-v1.schema.json:726]: Confirmed and fixed in 6e892ab. It appeared exactly once in the file, its own definition. `$defs/event` now carries two conditions: an `error` event must have an `error`, and an `auth.test` `result` event must have `data` matching `authTestResultData`. Proved by mutation: an undocumented `reason` now fails with `'made-up' is not one of ['granted', 'not-live', 'no-match']`, and removing `granted` fails with `'granted' is a required property`.

File: crates/irlume-cli/tests/machine_api.rs [hunk 4]
```diff
@@ -28,11 +28,101 @@ fn version_json_is_one_machine_document() {
...
+        .filter(|line| !line.trim().is_empty())
+        .map(|line| serde_json::from_str(line).expect("each line is a JSON document"))
+        .collect();
+    assert!(!lines.is_empty(), "fixture must contain events");
+
+    for (index, line) in lines.iter().enumerate() {
+        assert_eq!(line["sequence"], index as u64, "gapless from zero");
+        assert_eq!(line["contract_version"], 1);
+        assert_eq!(line["command"], "auth.test");
+        assert_eq!(line["operation_id"], lines[0]["operation_id"]);
```
[the remark this answers is not in the document]
Comment (archledger) [reply crates/irlume-cli/tests/machine_api.rs]: Confirmed and fixed in 6e892ab. An absent field indexes to `Value::Null`, so two missing ids compared equal and the test passed a stream carrying none. It now asserts `as_str()` is present, 32 characters, and all hex, then compares against the first line. This is the same defect as the Python check in the sibling thread, which is worth noting: one bug pattern, two languages, both silently passing.
